third_party/miniz-2.0.8/miniz.c

```diff
@@ -82,6 +82,20 @@ mz_ulong mz_adler32(mz_ulong adler, const unsigned char *ptr, size_t buf_len)
         }
         return ~crcu32;
     }
+#elif defined(USE_EXTERNAL_MZCRC)
```

Comment:
Should this be `#elif defined(USE_EXTERNAL_MZCRC) && USE_EXTERNAL_MZCRC`?

Reply:
Inside this specific file (as well as the .h, which widely uses ifdefs), defined(X) seems more idiomatic. This would be the only case of 'defined(X) && X' pattern, so it seems best to follow the rest of the file.